endoreg_db_api/settings_base.py

```
@@ -13,6 +13,7 @@
 from pathlib import Path
 import os
 import sys
+import yaml
```

Ruff (0.11.9)

`yaml` imported but unused

Remove unused import: `yaml`

(F401)